openshift/templates/issuer-agent/issuer-agent-deploy.overrides.sh

```diff
@@ -15,13 +15,21 @@ if createOperation; then
   readParameter "WALLET_KEY - Please provide the wallet encryption key for the environment.  If left blank, a 48 character long base64 encoded value will be randomly generated using openssl:" WALLET_KEY $(generateKey) "false"
   readParameter "WALLET_SEED - Please provide the indy wallet seed for the environment.  If left blank, a seed will be randomly generated using openssl:" WALLET_SEED $(generateSeed) "false"
   readParameter "WALLET_DID - Please provide the indy wallet did for the environment.  The default is an empty string:" WALLET_DID "" "false"
+  if [[ "$PROFILE" == "lsbc" ]]; then
+    # additional parameter required by lsbc deployment
+    readParameter "WEBHOOK_URL - Please provide the url to be used by the agent to send webhooks for the environment.  The default is an empty string:" WEBHOOK_URL "" "false"
```

Comment:
Make this prompt standard, there is no reason for it to be profile specific.

Reply:
The reason to NOT have it profile specific was that in use-cases like issuer-kit where the webhook URL is an internal service name (like when deploying `issuer-kit`) we likely don't want the person who is deploying the service having to figure out where to point it at.